database.js

```diff
@@ -17,10 +18,22 @@ function init() {
 
 // Make sure that the database schema is up to date
 function ensureVersion() {
-    const query = "SHOW COLUMNS FROM mentors LIKE 'pmsg'";
-    connection.query(query, (err, rows) => {
+    // Add the current version to the "version" table if it is empty
+    const checkVer = "SHOW COLUMNS FROM version LIKE 'ver'";
+    connection.query(checkVer, (err, rows) => {
         if (err) throw err;
         if (rows.length === 0) {
+            const addVer = "INSERT INTO version VALUES (?)";
+            connection.query(addVer, [package.version], (err) => {
+                if (err) throw err;
+            });
+        }
+    });
```

Comment:
I think we should add 1.0.2, not the current version, to the `version` table if it is empty. We should not assume that the table is up to date as is.